documentation/ES/chapter_1.md

```diff
@@ -1,34 +1,28 @@
+ Otros ejemplos relevantes de redes inalámbricas son las redes de área local inalámbrica WLAN (Wireless Local Area Network) y Bluetooth, siendo ampliamente utilizados en millones de productos que se interconectan entre sí o con aquellos de uso diario sirvendo de puente para estos a internet.
 
-Los sistemas radio celulares proporcionan un servicio de alta calidad que en ocasiones es comparable con el de los sistemas telefónicos cableados. En este tipo de redes, los encaminadores o estaciones base, son fijos y cableados y las unidades móviles se conectan y se comunican con su estación base más cercana dentro de su radio de alcance. 
+Para servir al objetivo de Locha Mesh, las redes inalámbricas con infraestructura son inviables y fisicamente vulnerables a los cambios repentinos que pueden afectar su infraestructura, como también sus limitaciones presentes en la misma y su cobertura de red, o las regulaciones mediadas por politicas restrictivas del libre uso de las redes, las cuales limitan al público el acceso a contenidos, también usado para censurar y perseguir tanto a los individuos como los contenidos solicitados por ellos, lo cual lleva nuestra investigación al segundo tipo de redes inalámbricas, sin infraestructura. 
 
-Cuando el dispositivo móvil sale del alcance de su estación base, se produce una transferencia o handoff desde la estación base inicial hasta una nueva, de tal modo que el dispositivo móvil es capaz de comunicarse de manera transparente a través de la red. Otros ejemplos relevantes de redes inalámbricas son las redes de área local inalámbrica (WLAN, Wireless Local Area Network).
+Las redes inalámbricas sin infraestructura no poseen encaminadores fijos, por tanto los integrantes de la red deben conectarse arbitrariamente entre sí. Es aquí donde encontramos las redes ad hoc, que surgen de manera espontánea, son dinámicas, y gozan por definición de capacidad de autoconfiguración. Este tipo de red permite interconectar nodos independientes, limitados en potencia de transmisión y procesamiento para proporcionar una mayor cobertura de red y capacidad de cómputo.
 
-En segundo lugar, se consideran las redes inalámbricas sin infraestructura. 
+Los componentes de estas redes, denominados nodos, funcionan como encaminadores que descubren y mantienen las rutas de otros nodos en la red, sin que exista un acuerdo previo en cuanto al papel que cada nodo debe asumir. Por el contrario, cada nodo toma sus decisiones autónomamente basándose en la situación de la red, por ejemplo, dos ordenadores personales equipados con tarjetas de red inalámbricas pueden configurar una red independiente siempre y cuando cada uno de los equipos este dentro del radio de cobertura del otro. 
```

Comment:
Los componentes de estas redes, denominados nodos, funcionan como encaminadores que descubren y mantienen las rutas de otros nodos en la red, sin que exista un acuerdo previo en cuanto al papel que cada nodo debe asumir. Por el contrario, cada nodo toma sus decisiones de forma autónoma basándose en la situación de la red, por ejemplo, dos ordenadores personales equipados con tarjetas de red inalámbricas pueden configurar una red independiente siempre y cuando cada uno de los equipos este dentro del radio de cobertura del otro.

Reply:
La palabra _"encaminador"_ no existe. Sustituir _encaminador_ por _enrutador_ 
> Enrutador: 1. tr. Inform. y Telec. Dirigir datos hacia una red

Reply:
agree, router is better than router :+1: